libraries/Network/src/NetworkServer.h

```diff
@@ -0,0 +1,62 @@
+/*
+  Server.h - Server class for Raspberry Pi
```

Comment:
Maybe we should update this line to reflect the actual description of this file. I think as long as the copyright and the license stays untouched it should be fine. (might be wrong though)